### DISCUSSION TITLE
Updating an existing compiled HTML+CSS GitHub Pages site - "unable to build page."

### DISCUSSION THREAD

**Discussion by AquaGeneral** [community/community#21554] [Other Feature Feedback, Questions, & Ideas]
<p>I’m having troubles with my site here: <a href="https://github.com/AquaGeneral/aquageneral.github.io" rel="nofollow noopener">https://github.com/AquaGeneral/aquageneral.github.io</a></p>
<p>It’s already been working for months, but as I try to update it now I get “Unable to build page” errors. I <a href="https://github.com/AquaGeneral/aquageneral.github.io/commit/2c608151d89021673540f513bc5901078c58288e" rel="nofollow noopener">also tried adding a “.nojekyll”</a> file and it’s still not working.</p>
<p>Note the site itself is compiled outside of this repository. This repo is just for hosting it on GitHub pages as simple HTML+CSS.</p>
<p>Any idea what is going wrong here?</p>

**Comment by foolmoron** [comment]
<p>Getting the same. I’m guessing something is wrong with github servers, not our code.</p>

**Comment by nalenb** [comment]
<p>Getting the same errors on 2 different sites</p>

**Comment by ghost** [comment]
<p>I’m looking at the site right-now, and it seems to be displaying properly. Is this the most recent pushed version? <br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/d/d7286c316716dac83edecf40fd550a9073bfe16a.png" data-download-href="/uploads/short-url/uHngcahbk2l91UmpwK5TCv1D5dU.png?dl=1" title="Screenshot 2017-11-03 at 9.50.29 pm.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/22669894/181065417-baac9f6e-d6f1-4dd5-bbc7-397d439ddfb8.png" alt="Screenshot 2017-11-03 at 9.50.29 pm.png" data-base62-sha1="uHngcahbk2l91UmpwK5TCv1D5dU" width="690" height="331" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/d/d7286c316716dac83edecf40fd550a9073bfe16a_2_690x331.png, https://aws1.discourse-cdn.com/github/optimized/2X/d/d7286c316716dac83edecf40fd550a9073bfe16a_2_1035x496.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/d/d7286c316716dac83edecf40fd550a9073bfe16a_2_1380x662.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/d/d7286c316716dac83edecf40fd550a9073bfe16a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot 2017-11-03 at 9.50.29 pm.png</span><span class="informations">1425×685 71.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by AquaGeneral** [comment]
<p>The key update I did was adding the v1.7 on my site. As you can see on the <a href="https://www.terrainformer.com/release-notes.html" rel="nofollow noopener">Release Notes</a> page they are still only showing v1.6’s release notes as the latest update.</p>

**Comment by birdcar** [comment, marked as answer]
<p>Sorry about that! This was actually an error on our end, and the team has already pushed a fix out for it, so it should be resolved.</p>
<p>With a generic build failure message like that, we would generally recommend checking out status page to see if there’s a known error:</p>
<p><a href="https://status.github.com/messages" rel="nofollow noopener">https://status.github.com/messages</a></p>
<p>And if you don’t see anything there, we’d recommend reaching out to the technical support team via our contact form so we can look into your repository specifically.</p>
<p><a href="https://github.com/support" rel="nofollow noopener">https://github.com/support</a></p>
